<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetUp</class>
 <widget class="QWidget" name="SetUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">QLabel{
	font: 20pt &quot;楷体&quot;;
	background-color: rgba(0,0,0,0);
	color: rgb(255, 255, 255);
	border-radius:4px;
}</string>
          </property>
          <property name="text">
           <string>设置</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="hideButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	width:20px;
	height:20px;
	border-radius:4px;
	color: rgb(255, 255, 255);
	font: 25pt;
}</string>
          </property>
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="addButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>添加</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="ListWidgetFloatWindow">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QListView::item {
    height: 50px;
}

QListView::item:hover {
	background:rgba(255,255,255,0.8);
}

QListView::item:selected { 
     background:rgba(255,255,255,0.8);
}

QPushButton {
	width:10px;
	height:10px;
	border-radius:4px;
}

QListWidget {
	background-color: rgba(255,255,255,0.4);
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
